dawn/test/unit-test/dawn/Optimizer/samples/TemporaryTypeUnstructured.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+##===-----------------------------------------------------------------------------*- Python -*-===##
+##                          _
+##                         | |
+##                       __| | __ ___      ___ ___
+##                      / _` |/ _` \ \ /\ / / '_  |
+##                     | (_| | (_| |\ V  V /| | | |
+##                      \__,_|\__,_| \_/\_/ |_| |_| - Compiler Toolchain
+##
+##
+##  This file is distributed under the MIT License (MIT).
+##  See LICENSE.txt for details.
+##
+##===------------------------------------------------------------------------------------------===##
+
+"""Generate input for StagerMerger tests"""
+
+import os
+
+import dawn4py
+from dawn4py.serialization import SIR
+from dawn4py.serialization import utils as sir_utils
+from google.protobuf.json_format import MessageToJson, Parse
+
+backend = "c++-naive-ico"
+
+def sparse_temporary():
+    outputfile = "DontDemoteSparse"
```

Comment:
To avoid confusion, I'd say it's better to give the same name to the output file and the script regenerating it. `DontDemoteSparse` sounds more evocative.